system_test/common/aest_db_SUITE.erl

```diff
@@ -95,9 +95,21 @@ format(Fmt, Args) ->
 
 node_spec(Name, DbHostPath) ->
     DbGuestPath = "/home/aeternity/node/data/mnesia",
-    aest_nodes:spec(Name, [], #{source  => {pull, "aeternity/aeternity:local"}, db_path => {DbHostPath, DbGuestPath}}).
+    aest_nodes:spec(Name, [], #{source  => {pull, "aeternity/aeternity:local"},
+                                db_path => {DbHostPath, DbGuestPath},
+                                genesis_accounts => genesis_accounts()}).
```

Comment:
I see that both [master](https://github.com/aeternity/aeternity/blob/831e1ee30e4092058ca35573bdfd8dd5f392c01d/rebar.config#L113) and [v1.4.0](https://github.com/aeternity/aeternity/blob/v1.4.0/rebar.config#L116) expect file at `data/aecore/.genesis/accounts_test.json` so macro [`EPOCH_GENESIS_FILE`](https://github.com/aeternity/aeternity/blob/831e1ee30e4092058ca35573bdfd8dd5f392c01d/system_test/common/helpers/aest_docker.erl#L34) does not need change - value `"/home/aeternity/node/data/aecore/.genesis/accounts_test.json"`. I also see path of node is `/home/aeternity/node` in both version (based on docker.md) so ok.